feature/search/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/search/SearchViewModel.kt

```diff
@@ -61,18 +61,15 @@ class SearchViewModel @Inject constructor(
                             flowOf(SearchResultUiState.EmptyQuery)
                         } else {
                             getSearchContentsUseCase(query)
-                                .asResult()
```

Comment:
Is it safe to remove this? (Don't we lose error handling, for instance?)

Reply:
The error handling is done the same way as in the asResult() function via the catch operator.

So basically the functionality is the same except of the onStart { Loading}